packages/design-system-react/src/components/blockies/README.mdx

```diff
@@ -3,7 +3,7 @@ import * as BlockiesStories from './Blockies.stories';
 
```

Comment:
Updating docs to align with other READMES in the DSR. We can align across the repo during our enablement sprint